rfc/rfc006-distributed-registry.md

```diff
@@ -279,6 +279,14 @@ When control over a DID document has been lost, the DID subject will have to hav
 Without Verifiable Credentials linked to the DID document, the DID document no longer has any value.
 The DID subject will have to go through the process of reacquiring all Verifiable Credentials for a new DID document.
 
+#### 3.3.4 Protection against public key zerging
```

Comment:
Is `key zerging` a thing? Could not find anything outside of game context.

Reply:
it has already made the urban dictionary: _Using massive amounts of weak, or generic units, to attack an enemy._

Reply:
I'm not sure whether a term has made it to Urban Dictionary is a good measure :'-D